omero/autogen_docs

```diff
@@ -51,6 +51,7 @@ for f in \
     omero-* \
     settings.env \
     setup_* \
+    docker_virtualenv*\
```

Comment:
We pick that term, I could not find a better one, I will review them post release. They are used in the server-centos6 installation